### PR TITLE
Bug 1132682 - Convert testMixedContentPage

### PR DESCRIPTION
**Pull request by galgeek**
https://bugzilla.mozilla.org/show_bug.cgi?id=1132682

Thanks @Ethcelon for all your work on PR #99!

This looks like the last of the mozmill security tests that's ready to convert, and I had a little time to work on it today.

### CODE REVIEW

File: firefox_ui_tests/remote/security/test_mixed_content_page.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                        favicon.value_of_css_property('list-style-image'))
+
+        self.identity_popup.box.click()
+
+        Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
+
+        self.assertEqual(self.identity_popup.encryption_label.text,
+                         self.browser.get_property('identity.broken_loaded'))
+
+        self.identity_popup.close()
```
Comment (chmanchester) [inline firefox_ui_tests/remote/security/test_mixed_content_page.py]: Nit: we already close this in teardown. This is independent of whether the test passes, we should just do it once in teardown.

File: firefox_ui_tests/remote/security/test_mixed_content_page.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+        self.identity_popup = self.browser.navbar.locationbar.identity_popup
+
+    def tearDown(self):
+        try:
+            self.browser.switch_to()
```
Comment (chmanchester) [inline firefox_ui_tests/remote/security/test_mixed_content_page.py]: This doesn't seem necessary, the test doesn't switch to any other window.

**Comment by galgeek** [comment]
Thanks for your feedback, @chmanchester!
I've updated the PR, and it's passing Travis.
Just let me know if there's anything else for this test.

**Comment by chmanchester** [comment]
Looking good. Just give it a squash and I'll get it merged.

**Comment by galgeek** [comment]
Squashed, and still passing Travis.

Thanks again, @chmanchester!

**Pull request merged**